### DISCUSSION TITLE
ISO 26262 certification for Zenoh

### DISCUSSION THREAD

**Discussion by break2make** [eclipse-zenoh/roadmap#167] [Q&A] [answered]
We found that Zenoh is getting popular in automotive domain (specifically SDV). The article (https://www.zettascale.tech/news/zettascale-demonstrates-zenoh-and-cyclone-dds-at-roscon/) from 2022, mentioned that zenoh will undergo the ISO 26262 certification. Is there any update on this?

**Comment by kydos** [comment, marked as answer]
Hello @break2make, 
  
    We are working on ISO-26262 certification, but this is part of the commercial offer from ZettaScale. If you are interested in learning more feel free to reach out. 

-- kydos

**Reply by break2make** [reply]
When will it be ISO-26262 certified? Any timeline?
